Neos.ContentRepository.Core/Classes/EventStore/EventNormalizer.php

```diff
@@ -85,11 +89,15 @@ public function __construct()
             RootNodeAggregateDimensionsWereUpdated::class,
             WorkspaceRebaseFailed::class,
             WorkspaceWasCreated::class,
+            WorkspaceWasRenamed::class,
             WorkspaceWasDiscarded::class,
             WorkspaceWasPartiallyDiscarded::class,
             WorkspaceWasPartiallyPublished::class,
             WorkspaceWasPublished::class,
-            WorkspaceWasRebased::class
+            WorkspaceWasRebased::class,
+            WorkspaceWasDeleted::class,
```

Comment:
Nitpick (sorry ;) ): WasDeleted => WasRemoved (we stick to "remove" instead of "delete" everywhere in the codebase for internal consistency)